### PR TITLE
Add blur on container

### PR DESCRIPTION
**Pull request by pboulch**
Add new optional option in shadowConfiguration to add blur effect on Containerview

### CODE REVIEW

File: BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift [hunk 1]
```diff
@@ -53,7 +53,11 @@ final class RootViewController: UIViewController {
         let viewController = ResizeViewController(initialHeight: 300)
         presentBottomSheetInsideNavigationController(
             viewController: viewController,
-            configuration: .default
+            configuration: BottomSheetConfiguration(
+                cornerRadius: 10,
+                pullBarConfiguration: .visible(.init(height: 50)),
```
Comment (mikhailmaslo) [inline BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift]: I'd expect `.default` appearance in Demo application, so can you revert this change?

**Review by mikhailmaslo — COMMENTED** [review]
Thank you for contribution!

File: BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift (hunk 1 above)
Comment (pboulch) [reply BottomSheetDemo/Sources/User Interface/Screens/Root/RootViewController.swift]: Yes, it's fixed

**Review by mikhailmaslo — APPROVED** [review]
